akka-stream/src/main/scala/akka/stream/impl/ActorProcessor.scala

```diff
@@ -201,7 +201,7 @@ private[akka] class SimpleOutputs(val actor: ActorRef, val pump: Pump) extends D
     }
   }
 
-  override def isClosed: Boolean = downstreamCompleted
+  override def isClosed: Boolean = downstreamCompleted && (subscriber ne null)
```

Comment:
this should not matter, but better safe than sorry. This will not allow the outputs to be considered closed while there is no subscriber.